src/ThemeProvider/ThemeProvider.tsx

```diff
@@ -35,205 +34,195 @@ declare module "@mui/material/styles" {
       };
     };
   }
+  interface Theme {
+    colorSchemes: {
+      light: ThemeOptions;
+      dark: ThemeOptions;
+    };
+  }
+  interface ThemeOptions {
+    colorSchemes?: {
+      light?: ThemeOptions;
+      dark?: ThemeOptions;
+    };
+  }
 }
 
-// theme defaults regardless of color mode. these are provided as an object that can be merged during color mode theme creation
-const defaultTheme: ThemeOptions = {
-  components: {
-    MuiAccordionSummary: {
-      styleOverrides: {
-        root: {
-          "&$expanded": {
-            marginBottom: -20
-          }
-        }
-      }
-    },
-    MuiDataGrid: {
-      styleOverrides: {
-        root: {
-          "& .MuiDataGrid-cell:focus, .MuiDataGrid-cell:focus-within, .MuiDataGrid-columnHeader:focus, .MuiDataGrid-columnHeader:focus-within":
-            {
-              outline: "none"
-            }
-        }
-      }
-    },
-    MuiFormLabel: {
-      styleOverrides: {
-        asterisk: {
-          color: "#d32f2f"
+// Define default components
+const defaultComponents = {
```

Comment:
`defaultComponents` added to share with both mode, previously it was defined golbally and there was no effect on this due to thw way MUI  design system works in MUI v6. If you provide both `colorSchemes` and `componnets`, the latter will override any styles defined in the former.